ng2-breadcrumb has been migrated to Angular 5 (ng5-breadcrumb)

aissam-ahbar opened the pull request.

Commit:
ng2-breadcrumb has been migrated to Angular 5. It is now called ng5-breadcrumb. The project is a fork from https://github.com/gmostert/ng2-breadcrumb

nielswitte commented on src/app/components/breadcrumb/breadcrumb.module.ts:
```diff
@@ -0,0 +1,20 @@
+  imports: [
+    CommonModule
+  ],
+  declarations: [BreadcrumbComponent]
```
You are missing the export for `BreadcrumbComponent`. Without this export the module won't work on its own. In your example `app.module.ts` you declare the component and service manually,

udayvunnam commented:
Issue after upgrading to ng5-breadcrumb and Angular 5.0

getting below error in console
compiler.js:466 Uncaught Error: Template parse errors:
Can't bind to 'useBootstrap' since it isn't a known property of 'breadcrumb'. ("

][useBootstrap]="false" prefix="Pepper Ops">
<div class="main-wrapper">
"): ng:///PagesModule/PagesComponent.html@30:16
'breadcrumb' is not a known element:

If 'breadcrumb' is an Angular component, then verify that it is part of this module.
To allow any element add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("
[ERROR ->]

aissam-ahbar committed:
Adding export to breadcrumb module.
Fixing an issue in getFriendlyNameForRoute leading to a compilation error with typescript.

aissam-ahbar commented:
Hi guys,

I fixed this issue coming from a missing export of the ng5-breadcrumb module. 

To download it, just use npm to install the ng5-breadcrumb:
https://www.npmjs.com/package/ng5-breadcrumb

Regards,
Aissam

evanjmg commented:
This doesn’t seem to work with the latest on npm

evanjmg commented:
I added the exports to the npm module, all my builds were fixed but no breadcrumbs appeared on my lazy loaded modules like before. Until this is fixed, I'm copying the source and importing the old ng2-breadcrumb directly into my  angular 5 project and if I can fix that aot error I'll make a pull to this repo. @aissamJ thanks for fixing the build issues, but I can't get the components to work. I think a very small pull request fixing the aot issues for angular 5 will suffice

evanjmg commented:
Upon adding the old ng2-breadcrumb directly into a massive sass project (30+ components use breadcrumbs lazily) without npm, everything works properly. It is simply the build system and exporting that needs reworking - not a complete rewrite or reformatting

Commit:
Adding configuration files for build and MIT LICENSE
